content_schemas/formats/popular_links.jsonnet

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
All the other jsonnet files start with `(import "shared/default_format.jsonnet") + {`. I'm not sure whether this is strictly necessary, but I think unless you've got a good reason to diverge it's best to be consistent.